capstone-ui/src/Components/ResultsBox/ResultsBox.jsx

```diff
@@ -10,6 +10,8 @@ export default function ResultsBox( { locationLevel, countyFIPS, selectedCandida
     let data = []
     let candidates = []
     const { stateName } = useParams();
+    const initialCandidates = 5;
```

Comment:
We can probably call this `pageSize` or `candidatesResultPageSize` given you are using this for initial candidates displayed and the increment of more candidates to show i.e. loading another page.